### PR TITLE
Log all low level messages

### PR DESCRIPTION
**Pull request by TrevorDecker**

### CODE REVIEW

**Comment by TrevorDecker** [comment]
@ramensandwich and @VivaanBahl can you check this

**Comment by VivaanBahl** [comment]
This is looking good to me, let's test it on the buggy today

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java [hunk 1]
```diff
@@ -200,6 +200,23 @@ public int peel(byte[] buffer, int start, int bytesAvailable)
 			encTicks = message.getDataWord() & 0xFFFF;
 			messagePubEnc.publish(estimateVelocity(message.getDataWord()));
 		}
+		else if (headerByte == RBSerialMessage.getHeaderByte("RBSM_MID_ERROR")) {
+			new RobobuggyLogicNotification("RBSM_MID_ERROR:"+message.getDataWord(), RobobuggyMessageLevel.EXCEPTION);
+		}
+		else if (headerByte == RBSerialMessage.getHeaderByte("RBSM_MID_ENC_RESET_REQUEST")) {
```
Comment (VivaanBahl) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java]: we never get this back from low level, this should be taken out

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java [hunk 2]
```diff
@@ -200,6 +200,23 @@ public int peel(byte[] buffer, int start, int bytesAvailable)
...
+			new RobobuggyLogicNotification("RBSM_MID_ENC_RESET_REQUEST:", RobobuggyMessageLevel.NOTE);
+		}
+		else if (headerByte == RBSerialMessage.getHeaderByte("RBSM_MID_ENC_RESET_CONFIRM")) {
+			new RobobuggyLogicNotification("RBSM_MID_ENC_RESET_CONFIRM:", RobobuggyMessageLevel.NOTE);
+		}
+		else if (headerByte == RBSerialMessage.getHeaderByte("RBSM_MID_MEGA_BRAKE_COMMAND")) {
```
Comment (VivaanBahl) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java]: This is also sent to low level, never received

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java [hunk 3]
```diff
@@ -200,6 +200,23 @@ public int peel(byte[] buffer, int start, int bytesAvailable)
...
+			new RobobuggyLogicNotification("RBSM_MID_MEGA_BRAKE_COMMAND:"+message.getDataWord(), RobobuggyMessageLevel.NOTE);
+
+		}		
+		else if (headerByte == RBSerialMessage.getHeaderByte("RBSM_MID_MEGA_STEER_COMMAND")) {
```
Comment (VivaanBahl) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java]: This is also sent to low level, never received

**Comment by VivaanBahl** [comment]
Tested on the buggy, will sign off and merge after the relevant comments have been addressed

**Comment by TrevorDecker** [comment]
has been ok'd

**Pull request merged**